python/generated/docstrings.h

```diff
@@ -230,10 +230,18 @@ static const char *__doc_bbp_sonata_PopulationProperties_alternateMorphologyForm
 
 static const char *__doc_bbp_sonata_PopulationProperties_biophysicalNeuronModelsDir = R"doc(Path to the template HOC files defining the E-Mode)doc";
 
+static const char *__doc_bbp_sonata_PopulationProperties_elementsPath =
+R"doc(Path to underlying elements H5 file. It is discouraged to directly
+access this.)doc";
```

Comment:
Maybe consider:

```
It is discouraged to directly access the contents of the file. Instead use 'libsonata' to read this file.
```

Personally, I would draw the line at manually opening/reading the file, rather than even knowing the path of the file that contains a particular population.

Reply:
yup, your wording is more clear, I will fix that.

(for future ref the `docstrings.h` is generated, so I hope you weren't reviewing it too carefully :))